### DISCUSSION TITLE
Available capacity --- and Consumed AmpHours 0Ah

### DISCUSSION THREAD

**Discussion by herrep** [Louisvdw/dbus-serialbattery#194] [Q&A]
Hi,

dbus-serialbattery connects with my JKBMS.  When viewing the parameters of dbus-serialbattery in Cerbo GX, I realized that the AmpHours parameter always shows 0Ah. The details page lists the installed capacity correctly, but the available capacity always says ---.

Is this the current behavior or gives this a hint for a wrong configuration on my end?

Best regards,
Peter

**Comment by lyncker** [comment]
same for me here... with dbus-spy inside the venus bash , I can see te correct values. ..

**Comment by mr-manuel** [comment]
Can you try to install the current `jkbms_ble` fork even, if you are not using Bluetooth? Currently it‘s the most updated one.

~~~bash

cd /tmp

wget https://github.com/Louisvdw/dbus-serialbattery/archive/refs/heads/jkbms_ble.zip
unzip jkbms_ble.zip

cp /tmp/dbus-serialbattery-jkbms_ble/etc/dbus-serialbattery/ /data/etc

chmod +x /data/etc/dbus-serialbattery/*.sh
chmod +x /data/etc/dbus-serialbattery/*.py
chmod +x /data/etc/dbus-serialbattery/service/run
chmod +x /data/etc/dbus-serialbattery/service/log/run

bash /data/etc/dbus-serialbattery/installlocal.sh
~~~
